### ISSUE TITLE
exam30/5/2023

### ISSUE THREAD

**Issue by Hoanggvien123** [Hoanggvien123/HoangVien_ExamTH2302026#1]
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Exam</title>
    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.9/angular.min.js"></script>
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.4/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
</head>
<body ng -app="myApp">
        <div ng-app="myApp" ng-controller="myCtrl">

            <div class="form-group row">
       
                <div class="col-xs-4">
                  <label for="ex3"></label>
                  <input class="form-control" ng-model="search" placeholder="enter text to search" type="text">
                  {{data}}
              </div>
              <div class="col-xs-4">
                <label for="ex3"></label>
                <input class="form-control" ng-model="categories" placeholder="search to categories" type="text">
               {{data}}
          </div>            
            </div>
            </div>
            
    <div ng-bind = "data">
        <div class="container-fluid">
            <div class="row">
                 <div class="col-sm-3">
                <dt>ASP.NET MVC</dt>
                <dd>Harrison Nick</dd>
                <p>ASP model view controller<br>
              design pattern,....</p>
                <button>Read more </button>
                </div>
          <div class="col-sm-3">
            <dt>Node.js</dt>
            <dd>Emarnuele DelBono</dd>
            <p>node.js is a wildly popular platform <br>for writing web application,... </p>
            <button>Read more</button>
          </div>
          <div class="col-sm-3">
           <dt>React.js</dt>
           <dd> Dimitri nester ruk</dd>
           <p>react is a javascript library that has  revolutionized how,....</p>
            <button>Read more</button>
          </div>
          <div class="col-sm-3">
            <dt>TypeScript</dt>
            <dd>Steve fenton </dd>
            <p>Microsoft TypeScript extend many familiar feature of ...</p>
            <button> Read more</button>
           </div>

        </div>
      </div>
    </div>
    
<script>
    var app = angular.module('myApp', []);
app.controller('myCtrl', function($scope) {
    
    $scope.data = "TypeScript"
        
})
var app = angular.module('myApp', []);
app.controller('myCtrl', function($scope) {
    
    $scope.data = "ASP.NET MVC"
        
});
</script>
</body>
</html>
